client/src/components/Predictions.js

```diff
@@ -1,23 +1,67 @@
 import React, { Component } from "react";
 import styled from "styled-components";
-import Row from "./Row";
+// import bennyboy from "../assets/profilePhotos/bennyboy.jpeg";
```

Comment:
why is this commented out ?

Reply:
tried to store the photos locally. didnt work, but will try again.